Cargo.toml

```diff
@@ -13,14 +13,15 @@ keywords = ["static", "site", "generator", "blog"]
 include = ["src/**/*", "LICENSE", "README.md"]
 
 [build-dependencies]
-clap = "2"
+clap = "3"
+clap_complete = "3"
```

Comment:
what does this do and where is it used?

Reply:
In v3 the completion functionality has been split off to the separate `clap_complete` package, I noticed that there is some completion code in build.rs that is commented out, so admittedly this is a bit preemptive.

I guess we could instead add it in a separate PR focused on bringing back completion